RevenueCatUI/CustomerCenter/Views/NoSubscriptionsView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  NoSubscriptionsView.swift
+//
+//
+//  Created by Andrés Boedo on 5/3/24.
+//
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct NoSubscriptionsView: View {
+
+    @Environment(\.dismiss) var dismiss
+    @State private var showRestoreAlert: Bool = false
+
+    var body: some View {
+        VStack {
+            Text("No Subscriptions found")
+                .font(.title)
+                .padding()
+            Text("We can try checking your Apple account for any previously purchased products")
```

Comment:
Nit: a more user-friendly string here might be something like "We can try checking your Apple account for any previous purchases" - I don't think most of my non-tech friends would know what a product means in this context